### PR TITLE
Optional empty message

### PR DESCRIPTION
**Pull request by moppymopperson**
This pull request adds a new property, `noEventsText` to `OCKCareCardViewController` and `OCKSymptomTrackerViewController`. If there are no events to display in the tableview, a label with the text in `noEventsTexts` will be displayed in the tableview's background view. If there are events to display or the `noEventsText` property has not been set, the tableview will behave exactly as it does now.

### CODE REVIEW

**Comment by kelseydedoshka-apple** [comment]
@moppymopperson - this is a great addition, thanks for contributing it back! If you would be willing to resolve the merge conflicts, or even consider adding this on the new OCKCareContentsViewController we would like to accept it.

**Comment by moppymopperson** [comment]
@kelseydedoshka-apple I'll see what I can do about the conflicts if I can find some free time this week. I'm not familiar with the new `OCKCareContentsViewController`, but I'll take a peek.

**Comment by moppymopperson** [comment]
That new `OCKContentCareViewController` is nice. I think people are really going to like it. I think I'll probably update my apps to take use it as well.

Merge should be good to go. I rebased my branch onto the latest master and made sure everything is still working.

**Comment by kelseydedoshka-apple** [comment]
@moppymopperson - glad you like the new view controller and thanks for getting your merge up to date. It looks like your PR now has some changes in it not regarding your fix - like all of the new assets and views. Are these intended to be changes?

**Comment by ninoguba** [comment]
I tried to use this but the noEventsText is not showing.  Guessing it is covered up by the page and header views. Still investigating...

File: CareKit/CareCard/OCKCareCardViewController.m [hunk 1]
```diff
@@ -108,6 +109,12 @@ - (void)viewDidLoad {
     _tableView.estimatedSectionHeaderHeight = 100.0;
     _tableView.sectionHeaderHeight = UITableViewAutomaticDimension;
     
+    _noDataLabel = [[UILabel alloc] init];
```
Comment (ninoguba) [inline CareKit/CareCard/OCKCareCardViewController.m]: This line should be the following for it to display correctly in the center of the view:

`UILabel *noDataLabel         = [[UILabel alloc] initWithFrame:CGRectMake(0, 0, _tableView.bounds.size.width, _tableView.bounds.size.height)];`

**Comment by ninoguba** [comment]
How you initialize the noDataLabel should be the following:

`UILabel *noDataLabel         = [[UILabel alloc] initWithFrame:CGRectMake(0, 0, _tableView.bounds.size.width, _tableView.bounds.size.height)];`

This is to make sure it will appear centered in the view and not covered by the header views of the Care Card, Symptom Tracker, and Care Contents

**Comment by ninoguba** [comment]
@moppymopperson I suggest you redo your PR with a new fork from master so that only your commits related to your Optional Empty State contribution will be included.  Thanks!

**Comment by moppymopperson** [comment]
@ninoguba Thanks for spotting that! Somehow I must've missed that line when I was cherry picking commits from my own master branch. 

Yeah, I think just cherry picking the relevant commits into a fresh branch will be more clear. I'll do that. There's no way to just change the branch a pull request points to is there? If not, I'll just make a new one and link to it from this one when I close it.

**Comment by moppymopperson** [comment]
Continuing in #193

**Pull request closed without merging**